test/recipe/admin/plugins/bakedComponent.ts

```diff
@@ -0,0 +1,12 @@
+export default defineNuxtPlugin({
+  name: "bakedComponent",
```

Comment:
this might be confused with `Baked.Component`. let's call this `baked`. it's likely to be the place where we put our other plugin requirements